spec/lib/secure_headers/headers/clear_site_data_spec.rb

```diff
@@ -20,30 +20,16 @@ module SecureHeaders
         name, value = described_class.make_header(true)
 
         expect(name).to eq(ClearSiteData::HEADER_NAME)
-        expect(value).to eq(normalize_json(<<-HERE))
-          {
-            "types": [
-              "cache",
-              "cookies",
-              "storage",
-              "executionContexts"
-            ]
-          }
-        HERE
+        expect(value).to eq(
+          ClearSiteData.make_header_value(ClearSiteData::ALL_TYPES)
+        )
       end
 
       it "returns specified types" do
         name, value = described_class.make_header(["foo", "bar"])
 
         expect(name).to eq(ClearSiteData::HEADER_NAME)
-        expect(value).to eq(normalize_json(<<-HERE))
-          {
-            "types": [
-              "foo",
-              "bar"
-            ]
-          }
-        HERE
+        expect(value).to eq(ClearSiteData.make_header_value(["foo", "bar"]))
```

Comment:
I'd prefer literals in the `eq`. It feels like you're comparing the output of the same function to itself.